packages/vincent-contracts/src/LibVincentDiamondStorage.sol

```diff
@@ -29,10 +29,10 @@ library VincentAppStorage {
     }
 
     struct AppStorage {
+        // TODO!: App id could be uint32
```

Comment:
I think the only issue with this would be that the OpenZeppelin EnumerableSet for uints uses `uint256`, so `numerableSet.UintSet permittedApps;` and `managerAddressToAppIds` would need to handle casting the `uint32`